metadata.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "theforeman-puppet",
+  "name": "desiderius-puppet",
```

Comment:
Sorry,  commit should not be paert of pull-up request; reverted.

--pk
________________________________

Subject: Re: [theforeman/puppet-puppet] Introduce parameter 'server_envs_target' that, when set, turns puppet's … (#422)


In metadata.json<https://github.com/theforeman/puppet-puppet/pull/422#discussion_r73171915>:

> @@ -1,10 +1,10 @@
>  {
> -  "name": "theforeman-puppet",
> +  "name": "desiderius-puppet",


wat? :)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/theforeman/puppet-puppet/pull/422/files/69f3ed2cb819cb9c46c426c801626b39522df6f1#r73171915>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AL6iuHYJOR7AsTWkFAm5GYNwfo4ETbXlks5qb1uEgaJpZM4JaEV9>.